das/doc.go

```diff
@@ -13,10 +13,10 @@ sufficiently likely that all block data is available as it is assumed that there
 are enough `light` availability instances active on the network doing sampling over
 the same block to collectively verify its availability.
 
-The central component of this package is the `DASer`. It performs one basic function:
-a sampling loop that performs DAS on new ExtendedHeaders in the network. The DASer kicks
-off this loop by loading its last DASed header (`checkpoint`) and kicking off a `catchUp`
-loop to DAS all headers between the checkpoint and the current network head. It simultaneously
+The central component of this package is the `samplingCoordinator`. It launches workers that
+perform DAS on new ExtendedHeaders in the network. The DASer kicks off this loop by loading
+its last DASed header (`checkpoint`) and kicking off worker pool
```

Comment:
Can you also make a note about the priority queue?

Reply:
Added line about recent headers prioritisation